File: lecture_002(Normal)/l_004_reversenumber.cpp
#include<iostream>
using namespace std;
int main(int args, char**argvs) {
    int num, rev=0;
    cout<<"Enter a number:";
    cin>>num;
    while(num != 0) {
        rev = rev*10 + num % 10;
        num /= 10;

    }
    cout<<"Reverse number is:"<<rev<<endl;
    
    return 0;
}

File: lecture_002(Normal)/l_004_reversenumber.java
import java.util.Scanner;
public class l_004_reversenumber {
    public static Scanner scn = new Scanner(System.in);
    public static void main(String[] args) {
        System.out.print("Enter a number:");
        int num = scn.nextInt();
        int rev=0;
        while (num != 0) {
            rev = rev*10 + num % 10;
            num /= 10;
        }
        System.out.println("Reverse number is:" + rev);
    }
}